documentation/staging/content/navigate/kubernetes/k8s-client-config.md

```diff
@@ -69,16 +69,16 @@ https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-g
 ```
 
 When running the application on Windows or Linux, add the appropriate directory to the `PATH` environment variable and make sure to
-run the application with that environment.  On MacOS, things are a bit more complicated.
+run the application with that environment.  On macOS, things are a bit more complicated.
 
-When running the WKT UI application on MacOS, the application inherits the environment of a daemon process called `launchd` instead
+When running the WKT UI application on macOS, the application inherits the environment of a daemon process called `launchd` instead
 of your environment.  By default, the `launchd` environment contains only a few core directories on the `PATH`
 (that is, `/usr/bin`, `/bin`, `/usr/sbin`, and `/sbin`).  This will cause `kubectl` invocations requiring access to one of
 the cloud providers' command-line tooling to fail if the tool is not found in one of those locations.  While it is
 possible for an administrative user to change the environment that `launchd` uses to address this issue, the application
 provides the `Extra Kubernetes Client Path Directories` field to explicitly add the directory where the cloud providers'
 command-line tooling is installed, to the `PATH` that the application uses to invoke `kubectl`.  Note that this field
-is only visible when running the application on MacOS.
+is only visible when running the application on macOS.
```

Comment:
This whole paragraph should be relocated since the `Extra Kubernetes Client Directories` was moved to the `Project Settings` page and renamed.